packages/radix-ui-themes/src/helpers/extract-props.ts

```diff
@@ -22,7 +22,7 @@ function mergePropDefs<T extends Record<string, PropDef>[]>(...args: T): Record<
 function extractProps<
   P extends { className?: string; style?: React.CSSProperties; [key: string]: any },
   T extends Record<string, PropDef>[]
->(props: P, ...propDefs: T): Omit<P, PropDefsWithClassName<T[number]>> {
+>(props: P, ...propDefs: T): Omit<P & { className?: string }, PropDefsWithClassName<T[number]>> {
```

Comment:
Related to https://github.com/radix-ui/themes/pull/295/files#diff-80f3b8c2c8536bb0a3e11e581238beb9653d2bb64044e714eec9fb573736579cL84